### ISSUE TITLE
Wrong version in installer

### ISSUE THREAD

**Issue by zarte13** [lukas-blecher/LaTeX-OCR#275]
Hi,
I had some trouble installing pix2tex GUI with `pip install pix2tex[gui]`
The installer installed tokenizers version 0.13.0 and it didn't work
And I got this error when running the latexocr command:
```
Traceback (most recent call last):
  File "C:\Python311\Lib\site-packages\transformers\utils\import_utils.py", line 1172, in _get_module
    return importlib.import_module("." + module_name, self.__name__)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1206, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1178, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1149, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "C:\Python311\Lib\site-packages\transformers\tokenization_utils_fast.py", line 25, in <module>
    import tokenizers.pre_tokenizers as pre_tokenizers_fast
  File "C:\Python311\Lib\site-packages\tokenizers\__init__.py", line 79, in <module>
    from .tokenizers import (
ImportError: DLL load failed while importing tokenizers: Le module spécifié est introuvable.

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Python311\Scripts\latexocr.exe\__main__.py", line 7, in <module>
  File "C:\Python311\Lib\site-packages\pix2tex\__main__.py", line 25, in main
    from .gui import main
  File "C:\Python311\Lib\site-packages\pix2tex\gui.py", line 18, in <module>
    from pix2tex import cli
  File "C:\Python311\Lib\site-packages\pix2tex\cli.py", line 22, in <module>
    from transformers import PreTrainedTokenizerFast
  File "<frozen importlib._bootstrap>", line 1231, in _handle_fromlist
  File "C:\Python311\Lib\site-packages\transformers\utils\import_utils.py", line 1162, in __getattr__
    module = self._get_module(self._class_to_module[name])
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python311\Lib\site-packages\transformers\utils\import_utils.py", line 1174, in _get_module
    raise RuntimeError(
RuntimeError: Failed to import transformers.tokenization_utils_fast because of the following error (look up to see its traceback):
DLL load failed while importing tokenizers: Le module spécifié est introuvable.
```
Then I tried installing the latest version of tokenizers (0.13.3) and overwrote the one in the installer and now it works fine
( Using `pip install tokenizers -U`)

**Comment by lukas-blecher** [comment]
Duplicate of #273 
Can you please confirm again that the model works with version 0.13.3?
Then I will update the requirements

**Comment by zarte13** [comment]
Yes it does work with version 0.13.3

**Comment by lukas-blecher** [comment]
Thanks, 
Fixed in 511fc437e5355a188be067b726d9fdf62ba43994
